승혁/CSS/Advanced_CSS/Animations_/Animations.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="kr">
```

Comment:
```suggestion
<html lang="ko">
```
"ko"는 한국어를 나타내는 언어 코드이고, "kr"은 대한민국을 나타내는 국가 코드입니다.